build/release.mk

```diff
@@ -26,13 +33,13 @@ release/deps: $(GORELEASER_BIN)
 .PHONY : release/build
 release/build: release/deps release/clean
```

Comment:
If we do this change, there is no point to split `release/build` and `release/pkg` targets. The only difference is the list of deps for this to targets.

Reply:
This is not my change. This line comes from the original branch (`ci/pre-release`)